scripts/initdb_and_run.sh

```diff
@@ -4,17 +4,18 @@ export LOCAL_DUMP="/app/scripts/prod_db_dump.sql"
 
 if [ ! -e /app/.cache/mysql/db.done ]; then
     echo "Initializing DB..."
-    python scripts/manage-db.py -d mysql://balrogadmin:balrogadmin@balrogdb/balrog create
     python scripts/import-db.py
 
     if [ -e "$LOCAL_DUMP" ]; then
       db_source="cat $LOCAL_DUMP"
     else
-      db_source="bunzip2 -c /app/scripts/sample-data.sql.bz2"
+      echo "Couldn't download production database dump. You must be connected to the internet the first time you start Balrog..."
```

Comment:
I was going to say that i doubted we'd ever show this message with `set -e` in use, as import-db.py will already have errored out, but it turns out it uses `exit()` rather than `exit(1)`. Supposing it did exit with non-zero status then this whole if/else block isn't needed any more, and we can do `cat $LOCAL_DUMP | mysql ....`

Reply:
Good idea - I'll fix import-db.py to exit better, and then we can simplify the shell code \o/